source/documents/insert.txt

```diff
@@ -92,6 +96,10 @@ To insert documents into your collection:
             .. figure:: /images/compass/insert-document2.png
               :figwidth: 590px
 
+            .. note::
```

Comment:
Thanks for remembering to do both tabs! Reminder to move the copy for this to a singe include file as discussed via Zoom.